src/components/Navigations/Sidedrawer/Side.js

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line no-unused-vars
+import React, { Fragment } from 'react';
+
+export default function Side(items) {
+  return (
+    <Fragment>
+      <ListGroup variant="flush">
+        {items.map(({ label, name, items: subItems, ...rest }) => (
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected a line break after this opening brace.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5592511)